### PR TITLE
bpo-27597: [doc] Add TracebackException usage examples

### PR DESCRIPTION
**Pull request by iritkatriel**

### CODE REVIEW

File: Doc/library/traceback.rst [hunk 1]
```diff
@@ -538,5 +548,67 @@ This last example demonstrates the final few formatting functions:
...
+With the helper classes, we have more options::
+
+   import sys
+   from traceback import TracebackException
+
+   def lumberjack():
+       bright_side_of_life()
+
+   def bright_side_of_life():
+       t = tuple((1,2,3))
```
Comment (markshannon) [inline Doc/library/traceback.rst:567]: `tuple` is redundant here. How about 
```
    t = "bright", "side", "of", "life"
    wayout = 5
    return t[wayout]
```
?

File: Doc/library/traceback.rst [hunk 2]
```diff
@@ -538,5 +548,67 @@ This last example demonstrates the final few formatting functions:
...
+   >>> TracebackException(*exc_info, limit=-2, capture_locals=True).print()
+   Traceback (most recent call last):
+     File "<stdin>", line 3, in lumberjack
+     File "<stdin>", line 4, in bright_side_of_life
+       t = (1, 2, 3)
+   IndexError: tuple index out of range
+
+   # We can change the format by replacing the default StackSummary class
+   >>> class CustomStackSummary(traceback.StackSummary):
+   ...     def format_frame(self, frame):
```
Comment (markshannon) [inline Doc/library/traceback.rst:593]: Shouldn't this be `format_frame_summary`?

File: Doc/library/traceback.rst [hunk 3]
```diff
@@ -538,5 +548,67 @@ This last example demonstrates the final few formatting functions:
...
+   >>> TracebackException(*exc_info, limit=-2, stack_summary_cls=CustomStackSummary).print()
+   Traceback (most recent call last):
+   <stdin>:3
+   <stdin>:4
+   IndexError: tuple index out of range
+   >>>
+
+   # We can omit frames from the output with a diffent StackSummary replacement
+   >>> class OmitLumberjack(traceback.StackSummary):
+   ...     def format_frame(self, frame):
```
Comment (markshannon) [inline Doc/library/traceback.rst:605]: ditto

**Review by markshannon — COMMENTED** [review]
I think there are a couple of errors in the examples.

**Comment by iritkatriel** [comment]
I don't know what this PR is. It seems to be targeting one of the branches of my fork.

**Comment by iritkatriel** [comment]
I made a new PR https://github.com/python/cpython/pull/125189. Some of the stuff here is out of date/irrelevant.

Thanks for digging this up.

**Pull request closed without merging**